### PR TITLE
[AlertDisplay] Added props to alert display

### PR DESCRIPTION
**Pull request by mehmetmalli**
## Hey, I just made a Pull Request!

I added optional alignment props to AlertDisplay component in order to allow custom location for alerts from the AlertAPI.

#### :heavy_check_mark: Checklist

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: packages/core-components/api-report.md [hunk 1]
```diff
@@ -51,8 +51,11 @@ import { Theme } from '@material-ui/core/styles';
 import { TooltipProps } from '@material-ui/core/Tooltip';
 import { WithStyles } from '@material-ui/core/styles';
 
-// @public
-export function AlertDisplay(_props: {}): JSX.Element | null;
+// Warning: (ae-forgotten-export) The symbol "AlertDisplayProps" needs to be exported by the entry point index.d.ts
```
Comment (benjdlambert) [inline packages/core-components/api-report.md:54]: Probably want to fix up these warnings too :)

**Review by benjdlambert — COMMENTED** [review]
Thanks for this! Just some nits!

**Comment by mehmetmalli** [comment]
@benjdlambert Hey, I just fixed those !

**Review by benjdlambert — APPROVED** [review]
Thanks :pray:

**Pull request merged**